Code/tests/test_autopkglib.py

```diff
@@ -0,0 +1,208 @@
+                            <key>requirement</key>
+                            <string>(identifier "com.google.Chrome" or identifier "com.google.Chrome.beta" or identifier "com.google.Chrome.dev" or identifier "com.google.Chrome.canary") and (certificate leaf = H"85cee8254216185620ddc8851c7a9fc4dfe120ef" or certificate leaf = H"c9a99324ca3fcb23dbcc36bd5fd4f9753305130a")</string>
+                        </dict>
+                    </dict>
+                </array>
+            </dict>
+            </plist>
+        """
+        )
+        self.munki_recipe = dedent(
+            """\
+            <?xml version="1.0" encoding="UTF-8"?>
+            <!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
+            <plist version="1.0">
+            <dict>
+                <key>Description</key>
+                <string>Downloads the latest Google Chrome disk image and imports into Munki.</string>
+                <key>Identifier</key>
+                <string>com.github.autopkg.munki.google-chrome</string>
+                <key>Input</key>
+                <dict>
+                    <key>NAME</key>
+                    <string>GoogleChrome</string>
+                    <key>MUNKI_REPO_SUBDIR</key>
+                    <string>apps</string>
+                    <key>pkginfo</key>
+                    <dict>
+                        <key>catalogs</key>
+                        <array>
+                            <string>testing</string>
+                        </array>
+                        <key>description</key>
+                        <string>Chrome is a fast, simple, and secure web browser, built for the modern web.</string>
+                        <key>display_name</key>
+                        <string>Google Chrome</string>
+                        <key>name</key>
+                        <string>%NAME%</string>
+                        <key>unattended_install</key>
+                        <true/>
+                    </dict>
+                </dict>
+                <key>MinimumVersion</key>
+                <string>0.2.0</string>
+                <key>ParentRecipe</key>
+                <string>com.github.autopkg.download.googlechrome</string>
+                <key>Process</key>
+                <array>
+                    <dict>
+                        <key>Arguments</key>
+                        <dict>
+                            <key>pkg_path</key>
+                            <string>%pathname%</string>
+                            <key>repo_subdirectory</key>
+                            <string>%MUNKI_REPO_SUBDIR%</string>
+                        </dict>
+                        <key>Processor</key>
+                        <string>MunkiImporter</string>
+                    </dict>
+                </array>
+            </dict>
+            </plist>
+        """
+        )
+
+    def tearDown(self):
+        pass
+
+    @patch("autopkglib.platform.platform")
```

Comment:
This only works now because `memoize` is now patched to return the function it's called with, rather than store the results and return the cache.